re.rizin.cutter.plugin.rz-ghidra.yml

```diff
@@ -0,0 +1,25 @@
+id: re.rizin.cutter.plugin.rz-ghidra
+branch: stable
+runtime: re.rizin.cutter
+runtime-version: stable
+appstream-compose: false
+sdk: org.kde.Sdk//5.15-21.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+build-options:
+  prepend-path: /usr/lib/sdk/llvm12/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
+  prefix: /app/extensions/rz-ghidra
+build-extension: true
+modules:
+  - name: rz-ghidra
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DBUILD_CUTTER_PLUGIN=ON
+      - -DCMAKE_INSTALL_PREFIX=/app/extensions/rz-ghidra
```

Comment:
```suggestion
```

Reply:
If I do not specify this path the installation happens on /app and I get an error because it is only readonly. Am I missing something ?

Reply:
the `prefix` build-options should have taken care of that.

Reply:
Removed the prefix definition at the beginning of the file

Reply:
it's actually the other way. You removed the wrong thing.